parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -19,8 +19,7 @@ fi
 export GPU_DEST=/usr/local/nvidia
 NVIDIA_DOCKER_VERSION=2.0.3
 DOCKER_VERSION=1.13.1-1
-NVIDIA_CONTAINER_RUNTIME_VER=2.0.0
-NVIDIA_DOCKER_SUFFIX=docker18.09.2-1
+NVIDIA_CONTAINER_RUNTIME_VER='3.5.0*'
```

Comment:
@cpuguy83 apt told me there was an even newer version so I updated to get it, and validated that it works

Reply:
Cool, this wasn't on the repo but tagged in GH when I was setting this up originally. 🎉